bindings/ios/MEGASdk.h

```diff
@@ -5400,6 +5401,32 @@ typedef NS_ENUM(NSInteger, AffiliateType) {
  */
 - (void)killSession:(uint64_t)sessionHandle;
 
+/**
+ * @brief Returns the deadline to remedy the storage overquota situation
+ *
+ * This value is valid only when MegaApi::getUserData has been called after
+ * receiving a callback MegaListener/MegaGlobalListener::onEvent of type
+ * MegaEvent::EVENT_STORAGE, reporting STORAGE_STATE_PAYWALL.
+ * The value will become invalid once the state of storage changes.
+ *
+ * @return Timestamp representing the deadline to remedy the overquota
+*/
+- (uint64_t)getOverquotaDeadlineTs;
```

Comment:
Please see this: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html

Use “get” only for methods that return objects and values indirectly.